dask_sql/physical/utils/statistics.py

```diff
@@ -87,21 +87,13 @@ def parquet_statistics(
     if not isinstance(layer, DataFrameIOLayer) or not isinstance(
         layer.io_func, ParquetFunctionWrapper
     ):
-        logger.warning(
-            f"Could not extract Parquet statistics from {ddf}."
-            f"\nAttempted IO layer: {layer}"
```

Comment:
I removed the warnings because since we're always trying to capture statistics now, this warning would come up every time the user works with a non-Parquet DataFrame. For example:
```
df = pd.DataFrame(
    {
        "a": [1.0] * 100 + [2.0] * 200 + [3.0] * 400,
        "b": 10 * np.random.rand(700),
    }
)
c.create_table("df", df)

result_df = c.sql("SELECT * FROM df")
```

Would throw a warning, which wouldn't make sense to the user.

Reply:
For simplicity we could try setting the level to `logger.info/logger.debug` which shouldn't be seen to the user unless they've set that specific logging level